src/components/ListTemplate/ColumnTemplates/MatchColumn.js

```diff
@@ -27,14 +27,19 @@ export const MatchColumn = memo(({
 }) => {
   const {
     existingRecordType,
-    existingStaticValueType,
     field,
-    fieldMarc,
-    fieldNonMarc,
+    matchDetails,
   } = record;
 
   const fieldSource = (field || existingRecordType || '').replace(/_/g, ' ');
-  const fieldMatched = (fieldMarc || fieldNonMarc || existingStaticValueType || '').replace(/_/g, ' ');
+  const matchRule = matchDetails[0];
+  const fieldsMatched = matchRule.existingMatchExpression.fields.flatMap(item => item.value);
```

Comment:
Many of those fields are not required in the schema. It means that they could be empty. Please use lodash.get function to avoid errors.

Reply:
Done